db/migrate/20220113195620_create_questions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateQuestions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :questions do |t|
+      t.string :title, null: false
+      t.boolean :for_partners, null: false, default: true
+      t.boolean :for_banks, null: false, default: true
```

Comment:
Great! Love to see the database constraints. This is really really important to build in whatever way you can to keep data consistent.